lib/galaxy/webapps/galaxy/controllers/library_common.py

```diff
@@ -1105,6 +1109,23 @@ def upload_dataset( self, trans, cntrller, library_id, folder_id, replace_datase
             elif upload_option == 'upload_directory':
                 uploaded_datasets, response_code, message = self.get_server_dir_uploaded_datasets( trans, cntrller, kwd, full_dir, import_dir_desc, library_bunch, response_code, message )
             elif upload_option == 'upload_paths':
+                uuid_list = kwd.get('uuid_list',None);
+                if(uuid_list != None):
+                    uuid_list = kwd.get('uuid_list',None).split('\n');
+                    kwd['uuid_list'] = uuid_list;
+                remote_dataset_type_list = kwd.get('remote_dataset_type_list',None);
```

Comment:
Probably it would be more Galaxy-ish to call these ``uuids`` instead of ``uuid_list``, ``remote_dataset_types`` instead of ``remote_dataset_type_list``, etc....